client/components/DateSelector/options.js

```diff
@@ -0,0 +1,52 @@
+import { DateUtils } from 'react-day-picker';
+
+const today = new Date();
+
+const oneMonthBack = DateUtils.addMonths(new Date(), -1);
+const sixMonthsBack = DateUtils.addMonths(new Date(), -6);
+const twelveMonthsBack = DateUtils.addMonths(new Date(), -12);
+const startOfThisYear = new Date(`1/1/${today.getFullYear()}`);
+const oneWeekBack = new Date(new Date().setDate(today.getDate() - 7));
+
+const getMonthsLongName = (...dates) => {
+  const months = dates.map(date => date.toLocaleString('default', {
+    month: 'long',
+  }));
+  return months.join(' - ');
+};
+
+const formatDateToMMDD = (...dates) => {
+  const localeStringWithoutYear = dates
+    .map(date => date.toLocaleDateString('en-Us').split('/'))
+    .map(dateArr => {
+      const [day, month] = dateArr;
+      return [day, month].join('/');
+    });
+
+  return localeStringWithoutYear.join(' - ');
+};
+
+const options = [
+  {
+    text: `Last Week ( ${formatDateToMMDD(today, oneWeekBack)})`,
```

Comment:
Swap the order of the arguments for `formatDateToMMDD` so the option text shows startdate - enddate instead of the other way around.

Reply:
Oh yeah. Good catch.

Reply:
> Looks excellent! Nice styles on the daypicker. I left some comments on the code and a few other things below.
> 
> * I'm getting several proptype errors from the Weekday component.
> * If `<DateRanges />` is expanded and the user opens the `<DayPicker />` by clicking the calendar, I think `<DateRanges />` should collapse.
> * Lets ask UI/UX about some of the labels in DateRanges.
> * We'll need to hook this up to redux but can be done in a different ticket/PR.
> * Some fonts need to be updated but no big deal. Can do this when we start adding several selectors to the menu.

- Added date picker close when dropdown opens and vice versa
- Changed the date range options to be consistent with the MMDD MMYYY formats.
- Adjusted font styles 
- Removed array-callback-return and consistent-return rules for linting. NEEDS APPROVAL. It didn't allow to have branched conditional return statement. Check lines 19 and 20 in [this file](https://github.com/hackforla/311-data/blob/830-FRONT-dateRangePicker/client/components/DateSelector/options.js)
- Didn't hook the component to the redux store. We can have another PR for it.

Reply:
`Removed array-callback-return and consistent-return rules for linting. NEEDS APPROVAL. It didn't allow to have branched conditional return statement. Check lines 19 and 20 in this file`

left comments in another review below